specs/Standards/MultiToken/Core.md

```diff
@@ -24,6 +24,21 @@ Prior art:
 - [ERC-1155]
 - [NEAR Fungible Token Standard][FT], which first pioneered the "transfer and call" technique
 - [NEAR Non-Fungible Token Standard][NFT]
+## Rationale
+
+Why have another standard, aren't fungible and non-fungible tokens enough?  The current fungible token and non-fungible token standards, do not provide support for performing batch transactions, representing many tokens in a single contract, as well as the flexibility to define different token types with different behavior in a single contract. This is something that makes it impossible to be interoperable with other major blockchain networks such as Ethereum.
+
+The standard here introduces a few concepts that evolve the original [ERC-1155] standard to have more utility, while maintaining the original flexibilty of the standard. So keeping that in mind, there is no explicit token type. Where the token type is set to be something that is FT or NFT or something in between.
```

Comment:
> So keeping that in mind, there is no explicit token type. Where the token type is set to be something that is FT or NFT or something in between.

I prefer to say something like, "This is a new token type: `<token type name>`. It combines two main features of FT and NFT: it allows to represent many token types in a single contract, and it's possible to store the amount for each token".

From the current wording, it's still unclear whether we try to put FT/NFT inside MT or it's a new token type.